.travis.yml

```diff
@@ -12,7 +12,6 @@ php:
 
 matrix:
   allow_failures:
-    - php: 7.1
```

Comment:
Ok I will fix this. 

But can you explain to me the usefulness of this `DISABLE_ASSERTIONS`? I cannot find it referenced anywhere in the source...